backend/coins/btc/maketx/txsize_internal_test.go

```diff
@@ -29,8 +29,7 @@ import (
 )
 
 func testEstimateTxSize(
-	t *testing.T, useSegwit bool, outputScriptType signing.ScriptType, changeScriptType *signing.ScriptType) {
-
+	t *testing.T, useSegwit bool, outputScriptType signing.ScriptType, changeScriptType signing.ScriptType) {
```

Comment:
Ah but now you don't need to specify the same type twice

Reply:
`... useSegwit bool, outputScriptType, changeScriptTypesigning.ScriptType)` which makes the line shorter.

Reply:
Fixed.